docs/plugins.yml

```diff
@@ -0,0 +1,78 @@
+---
+#####################
+# We're excited you wrote plugin! Here's how you share it with others:
+#
+# 1. Add a newline anywhere in the list below
+# 2. List your plugin following the <user>/<repo> format
```

Comment:
I would like to continue to sort by the name of the plugin, not the maintainer. In this particular markdown file, we will not include the maintainer in the name as part of the sort.